routes/index.js

```diff
@@ -5,6 +5,7 @@ app.use('/healthcheck', require('./healthCheck.js'))
 app.use('/auth', require('./auth.js'))
 app.use('/users', require('./users.js'))
 app.use('/members', require('./members.js'))
+app.use('/tasks', require('./tasks'))
```

Comment:
GET /tasks
POST /task (singular?)

Reply:
Yes, Makes sense. I have updated the PR.